views/partials/notLoadingNotice.html

```diff
@@ -0,0 +1,18 @@
+<div class="not-loading-notice">
+	<div class="o-message o-message--alert  o-message--neutral" data-o-component="o-message">
+		<div class="o-message__container">
+			<div class="o-message__content ">
+				<p class="o-message__content-main">
+					<span class="o-message__content-highlight">
+						Content not loading?
+					</span>
+					Subscribers can also read <a href="https://www.ft.com/content/{{id}}?referrer=amp-unreadable" rel="nofollow">{{title}}</a> on <a href="https://www.ft.com">ft.com</a>
```

Comment:
would `webUrl` be more appropriate?

Reply:
the `amp-unreadable` is a great idea btw

Reply:
I've updated this to use `webUrl` 👍